assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -200,39 +239,93 @@ const CheckoutEditor = ( { attributes, setAttributes } ) => {
 							) }
 						</Notice>
 					) }
+					<ToggleControl
+						label={ __(
+							'Show a "return to cart" link',
+							'woo-gutenberg-products-block'
+						) }
+						checked={ showReturnToCart }
+						onChange={ () =>
+							setAttributes( {
+								showReturnToCart: ! showReturnToCart,
+							} )
+						}
+					/>
+					{ showReturnToCart &&
+						( currentPostId !== CHECKOUT_PAGE_ID ||
+							cartPageId ) && (
```

Comment:
Is it intended that the page picker is displayed only if:

- The checkout block is on the Woo `checkout` page (at the time of editing).
- OR
- The user has previously set a cart page id.

In the case that the merchant has set up both woo `cart` & `checkout` pages with blocks, but previously was using shortcodes, and had set a `cartPageId` (to test blocks) .. I think the page picker would display? Probably an edge case, I'm thinking maybe we can don't need the `|| cartPageId`.

Reply:
> The checkout block is on the Woo checkout page (at the time of editing)

Backwards. Picker if the page is NOT the woo checkout page at time of editing. 

The picker displays because they have made a selection, so we continue to respect their choice. It avoids extra logic when reading that value during block rendering too - otherwise it would need to decide whether or not to use the option which is too much effort imo